ninja-core/src/main/java/ninja/template/TemplateEngineFreemarker.java

```diff
@@ -263,6 +264,9 @@ public void invoke(Context context, Result result) {
         map.put("reverseRoute", templateEngineFreemarkerReverseRouteMethod);
         map.put("assetsAt", templateEngineFreemarkerAssetsAtMethod);
         map.put("webJarsAt", templateEngineFreemarkerWebJarsAtMethod);
+        
+        map.put("authenticityToken", new TemplateEngineFreemarkerAuthenticityTokenDirective(context));
```

Comment:
we need a test to make sure authenticityToken and authenticityForm is added to the templating engine... We should really make sure that your great feature does not get destroyed by eg a bogus merge...

You can do that either via a unit test or in the ninja-integration-test project...